router/oauthResponseHandler/oauthResponseHandler.go

```diff
@@ -189,14 +189,14 @@ func (authErrHandler *OAuthErrResHandler) GetTokenInfo(refTokenParams *RefreshTo
 	if isRefreshActive, isPresent := authErrHandler.refreshActiveMap[refTokenParams.AccountId]; isPresent && isRefreshActive {
 		accountMutex.Unlock()
 		if refVal != nil {
-			token := refVal.Account.AccessToken
-			authErrHandler.logger.Debugf("[%s request] [Active] :: (Read) %s response received from cache(rt-worker-%d): %s\n", loggerNm, logTypeName, refTokenParams.WorkerId, token)
+			secret := refVal.Account.Secret
+			authErrHandler.logger.Debugf("[%s request] [Active] :: (Read) %s response received from cache(rt-worker-%d): %s\n", loggerNm, logTypeName, refTokenParams.WorkerId, string(secret))
 			return http.StatusOK, refVal
 		}
 		// Empty Response(valid while many GetToken calls are happening)
 		return http.StatusOK, &AuthResponse{
 			Account: AccountSecret{
-				AccessToken: "",
+				Secret: nil,
```

Comment:
Is it okay to have a string since if you marshal it anywhere else it would return null and not ""

https://github.com/golang/go/issues/17704#issuecomment-257434879

Reply:
Done